src/requests/payment-request.ts

```diff
@@ -3,10 +3,35 @@ import { getDetailedSessionById } from "./session";
 import { getLatestFlowGraph } from "../requests/flow";
 import keyPathAccessor from "lodash.property";
 import setByKeyPath from "lodash.set";
-import { findNextNodeOfType } from "../models/session/logic";
 import { Node, FlowGraph, ComponentType } from "../types";
 import type { PaymentRequest, Session, KeyPath, Value } from "../types";
 
+export class PaymentRequestClient {
+  protected client: GraphQLClient;
+
+  constructor(client: GraphQLClient) {
+    this.client = client;
+  }
+
+  async create(args: {
+    sessionId: string;
+    applicantName: string;
+    payeeName: string;
+    payeeEmail: string;
+    sessionPreviewKeys: Array<KeyPath>;
+  }): Promise<PaymentRequest> {
+    return createPaymentRequest(this.client, args);
+  }
+
+  async _markAsPaid(paymentRequestId: string): Promise<boolean> {
+    return _markPaymentRequestAsPaid(this.client, paymentRequestId);
+  }
```

Comment:
I'm introducing a pattern here to prefix methods which shouldn't normally be called. These are methods used by tests to manipulate state. The prefix is a strong signal that this shouldn't be used in normal business logic.

This convention does (again) clash with the convention of unused named variables and it's also a common convention for private variables but I think it works here since it's always part of a chained method call rather than a bare variable by itself. Feel free to suggest alternatives if you disagree.